File: C++/problems.cpp
#include <iostream>

int add(int a, int b) { return a + b; }
int subtract(int a, int b) { return a - b; }
int multiply(int a, int b) { return a * b; }
int divide(int a, int b) { return b != 0 ? a / b : 0; } // Add error handling for division by zero

int main() {
    int x = 10, y = 5;
    std::cout << "Add: " << add(x, y) << std::endl;
    std::cout << "Subtract: " << subtract(x, y) << std::endl;
    std::cout << "Multiply: " << multiply(x, y) << std::endl;
    std::cout << "Divide: " << divide(x, y) << std::endl;
    return 0;
}

File: KOTLIN/problems.kt
fun add(a: Int, b: Int): Int = a + b
fun subtract(a: Int, b: Int): Int = a - b
fun multiply(a: Int, b: Int): Int = a * b
fun divide(a: Int, b: Int): Int? = if (b != 0) a / b else null // Add error handling for division by zero

fun main() {
    val x = 10
    val y = 5
    println("Add: ${add(x, y)}")
    println("Subtract: ${subtract(x, y)}")
    println("Multiply: ${multiply(x, y)}")
    println("Divide: ${divide(x, y) ?: "Error (division by zero)"}")
}
